Source/Core/VideoBackends/OGL/BoundingBox.h

```diff
@@ -9,9 +9,15 @@ namespace OGL
 class BoundingBox
 {
 public:
-  static void Init();
+  static GLuint image_mask_fbo;
+  static bool image_mask_updated;
+  static bool image_mask_cleared;
+
+  static void Init(float xscale, float yscale);
   static void Shutdown();
 
+  static void SetEfbScaleChanged(float xscale, float yscale);
```

Comment:
Done. `PixelShaderManager::SetEfbScaleChanged` also needs to be changed at some point.